### DISCUSSION TITLE
Space ROS Technical Meeting Agenda 2023-12-14

### DISCUSSION THREAD

**Discussion by Bckempa** [space-ros/space-ros#117] [General]
Previous agenda: #115 

### Agenda
- [x] Review issues and PRs ([nominated for next release](https://github.com/orgs/space-ros/projects/3/views/6)).
- [x] Open next meeting discussion

**Comment by ivanperez-keera** [comment]
- [x] Introduce any new attendees.
- [x] Summarize any PRs merged since last meeting.
- [ ] Tag last docker image in docker hub via Github Actions.

**Reply by ivanperez-keera** [reply]
Another: IKOS is about to undergo a release freeze. Expected release date is end of December. There's a pending issue in Space ROS related to IKOS. If we have a documented way to build and test Space ROS on our side, then I'll be able to check that it's using the new version.

**Comment by Bckempa** [comment]
Opened next [meeting agenda for 2023-12-28](https://github.com/space-ros/space-ros/discussions/122)
